includes/express-checkout/class-wc-payments-express-checkout-button-display-handler.php

```diff
@@ -173,8 +173,8 @@ function( $js_config ) use ( $order ) {
 					$user_email = isset( $_POST['email'] ) ? sanitize_email( wp_unslash( filter_input( INPUT_POST, 'email' ) ) ) : $session_email;
 
 					$js_config['order_id']      = $order->get_id();
-					$js_config['pay_for_order'] = sanitize_text_field( wp_unslash( $_GET['pay_for_order'] ) );
-					$js_config['key']           = sanitize_text_field( wp_unslash( $_GET['key'] ) );
+					$js_config['pay_for_order'] = sanitize_text_field( wp_unslash( isset( $_GET['pay_for_order'] ) ) );
+					$js_config['key']           = sanitize_text_field( wp_unslash( isset( $_GET['key'] ) ) );
```

Comment:
`isset` returns a boolean. With the new changes, is this going to return the expected output?

Reply:
Yes, good point. Actually, I see that `isset` is checked for both params above in the beginning of the function so I'll just go ahead and add `phpcs:ignore`

Reply:
Sounds good, I agree 👍